grammar.pegjs

```diff
@@ -77,7 +77,7 @@ builtinCommandName
  = '[[' / '['
 
 argument "command argument"
- = commandName
+ = concatenation
```

Comment:
Arguments could be command names, but they are already covered by concatenation rule that, as I understand, match sequences of characters...